docs/se/guides/health.adoc

```diff
@@ -70,391 +69,199 @@ from the root directory of your project (helidon-quickstart-se).
 </dependency>
 ----
 
+Handling health checks is part of Helidon's observability support.
+By default, when you add the dependency for the built-in health checks, Helidon automatically registers the built-in checks.
+But the generated project does otherwise to illustrate how you can tailor its handling of health checks as you see fit.
```

Comment:
[question for clarity - not a change request] I read this to mean that the generated project does _not_ automatically register the built-in checks allowing you to configure the handling of health checks as needed.

Reply:
Basically, yes.

I will change this phrasing to clarify what I'm trying to say. (I'm pushing another commit anyway to address the earlier comment.)